lib/resources/variant.rb

```diff
@@ -0,0 +1,44 @@
+module MusicTodayApiWrapper
+  module Resources
+    class Variant
+      attr_accessor :sku,
+                    :price,
+                    :quantity_available,
+                    :size,
+                    :format,
+                    :bitrate
+
+      # rubocop:disable ParameterLists
+      def initialize(sku, price, quantity_available, size = nil, format = nil,
+        bitrate = nil)
+        @sku = sku
+        @price = price
+        @quantity_available = quantity_available
+        @size = size
+        @format = format
+        @bitrate = bitrate
+      end
+
+      def self.from_hash(variant_hash)
+        size, format, bitrate = nil
+
+        variant_hash['variantNames'].each do |name|
```

Comment:
Better send the variants formatted in a hash to facilitate manipulation of multiple variants on the client.